fdbclient/NativeAPI.actor.cpp

```diff
@@ -3915,6 +3896,50 @@ Future<Void> getWatchFuture(Database cx, Reference<WatchParameters> parameters)
 	return Void();
 }
 
+namespace {
+
+// NOTE: Since an ACTOR could receive multiple exceptions for a single catch clause, e.g. broken promise together with
```

Comment:
I'm not sure I understand this. A catch clause should only see one error at a time

Reply:
I am not sure about this, but I am seeing multiple exceptions causing the reference count went negative.